tests/test_cloudformation/test_cloudformation_custom_resources.py

```diff
@@ -0,0 +1,121 @@
+import boto3
+import json
+import sure  # noqa # pylint: disable=unused-import
+import time
+
+from moto import mock_lambda, mock_cloudformation, mock_logs, mock_s3, settings
+from pprint import pprint
+from unittest import SkipTest
+from uuid import uuid4
+from tests.test_awslambda.utilities import wait_for_log_msg
+from .fixtures.custom_lambda import get_template
+
+
+def get_lambda_code():
+    pfunc = """
+def lambda_handler(event, context):
+    print(event)
+    response = dict()
+    response["Status"] = "SUCCESS"
+    response["StackId"] = event["StackId"]
+    response["RequestId"] = event["RequestId"]
+    response["LogicalResourceId"] = event["LogicalResourceId"]
+    response["PhysicalResourceId"] = "{resource_id}"
+    response_data = dict()
+    response_data["info_value"] = "special value"
```

Comment:
This is the meat of what we're actually testing here.
Can this value be persisted via the `cfnresponse`-module, and picked up via boto3's `describe_stacks`-request?

(Assume another call to a third-party was made to create a resource, and this `special value` was returned by the third-party.)